### DISCUSSION TITLE
No package libncurses-devel available

### DISCUSSION THREAD

**Discussion by rubikstriangle** [AcademySoftwareFoundation/OpenRV#60] [Build Issues] [answered]
I just installed  fresh copy of centOS 7.9 and now I'm attempting to install the dependencies for installing OpenRV.  There were a few packages that were not available until I installed epel-release.  Now I'm left with just libncurses-devel as not available and my googling skills have failed to turn up an answer.   It seems like this might be a very old package that was replaced.  Has it been supplanted by ncurses-devel?

**Comment by rogernelson** [comment]
Hi @rubikstriangle Have you had a chance to take a look at the build guide for CentOS [here](https://github.com/AcademySoftwareFoundation/OpenRV/blob/main/doc/build_system/config_linux_centos7.md)? The `yum` command there should install all the packages you need (including ncurses):
> sudo yum install alsa-lib-devel autoconf automake avahi-compat-libdns_sd-devel bison bzip2-devel cmake-gui curl-devel flex glew-devel libXcomposite libXi-devel libaio-devel libffi-devel libncurses-devel libtool libxkbcommon openssl-devel pulseaudio-libs pulseaudio-libs-glib2 ocl-icd opencl-headers python3 python3-devel qt5-qtbase-devel readline-devel sqlite-devel tcl-devel tk-devel yasm zlib-devel

**Reply by rubikstriangle** [reply]
I didn't see this instruction set but it looks like it is essentially the same as the outline in the .md file that lives here:
OpenRV/doc/build_system/config_linux_centos7.md

Both refer to a libncurses-devel that doesn't seem to be available for centos 7.9, at least not in any of the repositories that I have tried and google isn't of much help here.

**Reply by rubikstriangle** [reply]
I was able to complete the build without any noticeable error from the missing library

**Comment by tjjackson** [comment, marked as answer]
I believe this is a typo and was meant to be ncurses-devel

**Comment by geffrak** [comment]
I created this to fix the typo: https://github.com/AcademySoftwareFoundation/OpenRV/pull/83 Thanks all!
